GoogleForCreators/web-stories-wp
# Media: Move list of transcodable mime types to JS

## Context

Moves transcodable mime types out of PHP / config and into a constants file 

See: https://github.com/GoogleForCreators/web-stories-wp/issues/10591#issue-1140124787

## Summary

Moves the mime types into `packages/story-editor/src/app/media/index.js`  + updates code to pull from that location vs pulling from the config.

## Relevant Technical Choices



## To-do

N/A

## User-facing changes

N/A

## Testing Instructions

@spacedmonkey will test the VideoPress / JetPack code see:
https://github.com/GoogleForCreators/web-stories-wp/pull/10604#issuecomment-1046830698






This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---
...starting with JS files will add removal of PHP code soon.



Fixes #10591
